viewing_party/party.py

```diff
@@ -0,0 +1,129 @@
+#wave 1
+
+def create_movie(title, genre, rating):
+    if title ==  None:
+        return None
+    if genre == None:
+        return None
+    if rating == None:
+        return None
+    else:
+        movie = {"title": title, "genre": genre, "rating": rating}
+        return movie
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if movie['title'] == title:
+            user_data['watchlist'].remove(movie)
+            user_data["watched"].append(movie)
+        if user_data['watched'] == []:
+            user_data["watched"].append(movie)
+
+    return user_data
+
+#wave 2 pt 1 
+
+def get_watched_avg_rating(user_data): 
+    if len(user_data["watched"]) == 0:
+        return 0 
+    else:
+        ratings = []
+        for movie in user_data['watched']:
+            ratings.append(movie["rating"])
+        average = sum(ratings)/len(ratings)
+        return average 
+
+ # wave 2 pt 2
+ 
+def get_most_watched_genre(user_data):
+    genres_watched = []
+
+    if len (user_data["watched"]) == 0:
```

Comment:
careful with too many spaces! len is a function, and it's being called here. so keep the `len` and the `(` together! `len().` it won't break anything if you don't, but it is standard practice